feedback/static/css/feedback.css

```diff
@@ -40,6 +40,132 @@
     word-break: break-word;
 }
 
+/* Feedback-modal-window */
+.feedback_modal {
+    display: none;
+    position: fixed;
+    z-index: 9999;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+}
+
+.feedback_modal_overlay {
+    background-color: rgba(0, 0, 0, .4);
+    width: 100%;
+    height: 100%;
+    z-index: 9999;
```

Comment:
Do we need to have `z-index: 9999;` because we already have the same z-index for the wrapper block?